tagstudio/src/qt/modals/tag_search.py

```diff
@@ -115,19 +119,23 @@ def update_tags(self, query: str = ""):
             l = QHBoxLayout(c)
             l.setContentsMargins(0, 0, 0, 0)
             l.setSpacing(3)
+
             tw = TagWidget(self.lib, self.lib.get_tag(tag_id), False, False)
+
+            tag_colors = self.lib.get_tag(tag_id).color
             ab = QPushButton()
+            ab.setCheckable(True)
             ab.setMinimumSize(23, 23)
             ab.setMaximumSize(23, 23)
             ab.setText("+")
             ab.setStyleSheet(
                 f"QPushButton{{"
-                f"background: {get_tag_color(ColorType.PRIMARY, self.lib.get_tag(tag_id).color)};"
+                f"background: #d2d2d2;"
```

Comment:
maybe use a sensible name instead of the magic value?
`LIGHT_GRAY` as an example.